Plugin/CppApi/source/ArcGISCompassController.cpp

```diff
@@ -48,10 +51,17 @@ ArcGISCompassController::ArcGISCompassController(QObject *parent):
   });
 }
 
+/*!
+   \brief The destructor.
+ */
 ArcGISCompassController::~ArcGISCompassController()
 {
 }
 
+/*!
+   \property ArcGISCompassController::heading
```

Comment:
I don't think we need the \property tag?

Reply:
`\property` documents the getter, the setter, and the signal in one go.